src/Entity/Item.php

```diff
@@ -208,7 +208,7 @@ public function getType() {
 	 * @return Statement
 	 */
 	public function newClaim( Snak $mainSnak ) {
-		return new Statement( new Claim( $mainSnak ) );
```

Comment:
If this is a pull request that merges master into 3.x, how is it possible these changes pop up here? Something is still messed up, I'm afraid.